lib/tasks/pact.rake

```diff
@@ -1,5 +1,17 @@
 return if Rails.env.production?
 
 require "pact/tasks"
-require "pact_broker/client/tasks"
 require "pact/tasks/task_helper"
+
+desc "Verify that Collections honours all contracts in the pactfile created by a given branch of gds-api-adapters"
+task "pact:verify:branch", [:branch_name] => :environment do |t, args|
+  abort "Please provide a branch name. eg rake #{t.name}[my_feature_branch]" unless args[:branch_name]
+
+  pact_version = args[:branch_name] == "master" ? args[:branch_name] : "branch-#{args[:branch_name]}"
+
+  ClimateControl.modify GDS_API_ADAPTERS_PACT_VERSION: pact_version do
```

Comment:
Is this the right ENV to use? It looks like you're defining `GDS_API_ADAPTERS_PACT_BRANCH` but not `GDS_API_ADAPTERS_PACT_VERSION`, so I'd imagine this doesn't work unless there's some Jenkins magic happening under the hood 🤔

Reply:
https://github.com/alphagov/collections/pull/2297